common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -573,7 +573,7 @@ def available_translations(self, transcripts, verify_assets=True):
 
         # If we're not verifying the assets, we just trust our field values
         if not verify_assets:
-            translations = list(other_lang)
+            translations = list(other_lang) if other_lang else []
             if not translations or sub:
                 translations += ['en']
             return set(translations)
```

Comment:
Why are we converting `other_lang` (from whatever its original type is) to a `list`, and then back to a `set`?  Unless I'm missing something, this would be improved by creating a `set` right away.